docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+# Introduction to ERC721 (NFT) Smart Contracts
+
+This tutorial will start you with a basic ERC721 (NFT) smart contract on the Avalanche Network, regardless of your previous development experience. We will deploy our NFT on the Avalanche Fuji Testnet and view it on the Snowtrace Testnet Explorer. Note that these are not transferable to the Mainnet. However, once you feel comfortable launching your project, you can do so on Avalanche Mainnet and list it on an NFT marketplace.
+
+The following tools will be used during this tutorial:
+- [Pinata](https://www.pinata.cloud/): To store your NFT images and metadata.
+- [OpenZeppelin’s Wizard](https://wizard.openzeppelin.com/): to create the ERC721 smart contract.
+- [Remix IDE](https://remix-project.org/): To edit the code and deploy it to Fuji.
+- [Avalanche Testnet Faucet](https://faucet.avax.network/): To fund the deployment. 
+- [MetaMask browser Extension](https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn): To process transactions related to funding and deploying the smart contract.
+- [Snowtrace Testnet Explorer](https://testnet.snowtrace.io/): To view the deployed smart contract.
+
+
+:::caution
+**DISCLAIMER: This Solidity smart contract tutorial is for demonstration purposes only. Users should consider proper precautions, error handling, and safeguards for production use. No one at Ava Labs is responsible for your development, and you must take full responsibility for ensuring your code is secure.**
+:::
+
+## Preparing Your NFT Files
+
+The first step of setting up an NFT smart contract is having your NFT files ready to use. In this example, the files will get uploaded to [Pinata](https://www.pinata.cloud/), a pinning service that prevents files from being garbage collected on IPFS.
+
+If you are unfamiliar with the process of uploading image and metadata files to an IPFS provider for NFT collection usage, please check out [this article](preparing-nft-files.md). Ensure that your files are uploaded and your base URI is ready to plug into your smart contract.
+
+Once the image and metadata files are ready, we can prepare to deploy a smart contract.
+
+## Preparing Your Environment
+
+### MetaMask Extension
+
+You will need the MetaMask Extension installed on whatever browser you are using to be able to fund the deployment of the smart contract. If you have not done so already, download MetaMask and [add the Fuji network to MetaMask](https://support.avax.network/en/articles/6224787-how-to-connect-metamask-to-the-fuji-testnet). Create or import a Fuji account as necessary.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "...whatever browser you're using..." instead of "...whatever browser you are using..." and "If you've not done so already" instead of "If you have not done so already"